ui-v2/app/services/clipboard/local-storage.js

```diff
@@ -0,0 +1,28 @@
+import Service from '@ember/service';
+import { get } from '@ember/object';
+
+import Clipboard from 'npm:clipboard';
```

Comment:
Not sure if a rebase will catch this or not, but now that you have switched to auto-import, these `npm:` prefixes should go away.

Reply:
Yup, definitely. The standard javascript importing stuff is in my `ui-staging` branch though, which is basically the 'next'/1.6 branch, so won't be landing for a few weeks yet. I'm aiming for the bugfix that this is related to to land in 1.5.2 next week.

Once 1.5.2 has gone out I can rebase and amend this, I'm assuming it will scream at me when I try to build this after a rebase.